layouts/partials/posts/math.html

```diff
@@ -22,6 +22,10 @@
     });
   </script>
 {{- end -}}
+{{ if and (not .Params.disable_mathjax) (or (in (string .Content) "\\") (in (string .Content) "$")) }}
```

Comment:
MathJax is already here. Look at the previous lines.
But, maybe we should review it.
Could you remove this part by now?